runtime/src/nft/mod.rs

```diff
@@ -159,7 +160,12 @@ impl<T: Trait>
 
         // Insert into storage
         AccountForAsset::<T>::insert(registry_id, token_id, owner_account);
-        Assets::<T>::insert(registry_id, token_id, asset_info);
+
+        //TODO(mig) Remove once we have storage fee for metadata
+        let empty_info: <T as Trait>::AssetInfo = AssetInfo {
```

Comment:
@vedhavyas if you have a better idea to create an empty value, let me know.

Reply:
You can use `default` since its already a trait implemented for AssetInfo trait

Reply:
thanks I will try it out